README.md

```diff
@@ -1,2 +1,27 @@
-# stencil-select
-Stencil's fancy select component
+
+# Stencil Select
+
+A styleable, accessible `<select>` component.
+
+[Link to demo](#)
```

Comment:
Sorry – readme is also placeholder at the moment and shouldn’t be seriously reviewed yet.